package.json

```diff
@@ -24,7 +24,9 @@
     "bluebird": "3.3.4",
     "cross-spawn-async": "2.2.1",
     "dateformat": "1.0.12",
-    "debug": "2.2.0"
+    "debug": "2.2.0",
+    "firefox-launch": "git://github.com/shane-tomlinson/firefox-launch#3bd641d04a277ce1b865e6de4b4a568c04481326",
```

Comment:
@shane-tomlinson is it possible to publish this module? (so it is not `git://`. Users install this module on messed up computer setups and I don't trust npm to work properly with `git://` urls on all machines.

Reply:
ping @shane-tomlinson